components/Chips/tests/unit/ChipsTests.m

```diff
@@ -294,6 +294,39 @@ - (void)testPointInsideWithCustomHitAreaInsets {
                         withEvent:nil]);
 }
 
+- (void)testRemoveChipsManually {
```

Comment:
I think this test would be less implementation-specific if you only verify that the X origin of the placeholder when the Chip is remove is less than when a Chip is present.  Instead of verifying that adding and removing a Chip results in exactly the same X position.